src/main/resources/config.yml

```diff
@@ -122,6 +122,39 @@ bot:
     roles:
       - "000000000000000000:000000000000000000"
 
+    # A list of options that allow you to notify in a Discord Channel if an account has been unlinked/unlinked
+    notify_linkage:
```

Comment:
Let's rename this simply to "notify" (since it's already within the "linking" category)